Settings.h

```diff
@@ -17,15 +17,28 @@ in the source distribution for its full text.
 
 #define DEFAULT_DELAY 15
 
+typedef enum HeaderFormat_ {
```

Comment:
Better co-locate with the formatWidths array for easier maintainability, see comment above